### PR TITLE
index.md in 'u'

### LINKED ISSUE #15: Create index.md in `u`
Use the template found at https://docs.google.com/document/d/1Qqsge3TYgq1DM4DyAT4vbTYQ8RTT3COtU9dOoIBZMhY/edit#heading=h.rwm7jwhbzloe

… and create the file at `u/index.md`

### PR DESCRIPTION
**Pull request by voromahery**
Fixes #15 

Hello team,

We have made a new file in the folder 'U'.
Please,  have a look.

Thanks,
Ainiel

### CODE REVIEW

**Comment by ndorfin** [comment]
Please change your PR title to reflect which folder you created

File: u/index.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# List of artists starting with U:
+
+  Oops! It seems as if there isn't any Malagasy artist's name start with 'U'.
```
Comment (ndorfin) [inline u/index.md]: This looks good!
I'd suggest a rewording though:

```
Oops! It seems there aren't any Malagasy artists with names starting with 'U'.
```

**Review by ndorfin — CHANGES_REQUESTED** [review]
Almost there. Contact me for a re-review when you've made the fixes.

**Comment by ndorfin** [comment]
Also, it would be very cool if your PR description started with:

`Fixes #15 `

**Review by Ainatiana — APPROVED** [review]
That's  true Ainiel! If only Ulrich (Me) was a singer, there would be one.
Thanks man.
Good job!

File: u/index.md (hunk 1 above)
Comment (voromahery) [reply u/index.md]: Could you check it now, please?

Comment (voromahery) [reply u/index.md]: How about #32, because #15 is already existed.

**Pull request merged**